rest-api-spec/src/main/resources/rest-api-spec/api/indices.put_index_template.json

```diff
@@ -5,6 +5,8 @@
       "description":"Creates or updates an index template."
     },
     "stability":"stable",
+    "visibility":"feature_flag",
+    "feature_flag":"es.itv2_feature_enabled",
```

Comment:
These APIs (the index template V2 stuff) is no longer behind a feature flag, see: https://github.com/elastic/elasticsearch/pull/56123

Reply:
Updated, thanks @dakrone